bears/r/FormatRBear.py

```diff
@@ -15,10 +16,45 @@ class FormatRBear:
     LANGUAGES = "R"
 
     @staticmethod
-    def create_arguments(filename, file, config_file):
+    def create_arguments(filename, file, config_file,
+                         r_keeps_comments: bool = True,
+                         r_keeps_blank_lines: bool = True,
+                         r_brace_to_newline: bool = False,
+                         r_arrow_to_equal: bool = False,
+                         tab_width: int = SpacingHelper.DEFAULT_TAB_WIDTH,
+                         r_max_expr_length: int = 20):
+        """
+        :param r_keeps_comments:       Use to determine whether comments are
+                                       kept or not.
+        :param r_keeps_blank_lines:    Use to determine whether blank lines
+                                       are kept or not.
+        :param r_brace_to_newline:     Whether to put the left brace
+        :param r_arrow_to_equal:       Whether to replace the assign
+                                       operator "=" with "<-" or vice versa.
+        :param tab_width:              Number of space for indentation.
+        :param r_max_expr_length:      Maximum number of characters for an
+                                       expression.Note (different from the
+                                       maximum number of characters for a line).
+        """
+        def _map_to_r_bool(py_bool):
+            return 'TRUE' if py_bool else 'FALSE'
```

Comment:
please move that one outside of this function, better performance + we are python, we are adults :)